app/uk/gov/hmrc/nisp/views/includes/nirecordtaxyear.scala.html

```diff
@@ -86,7 +88,13 @@
                     classThreePayableWithPenalty <- niRecordTaxYear.classThreePayableByPenalty
                 ){
                         <p class="contributions-header">@Messages("nisp.nirecord.gap.youcanmakeupshortfall")</p>
-                        <p>@Html(Messages("nisp.nirecord.gap.payvoluntarycontrib",NispMoney.pounds(niRecordTaxYear.classThreePayable.getOrElse(0)),Dates.formatDate(classThreePayableWithPenalty.localDate),Dates.formatDate(classThreePayableByDate.localDate)))</p>
+
+                        @if(niRecordTaxYear.checkCutOffDate(Some(classThreePayableByDate),new LocalDate())) {
```

Comment:
I am sure that there is a way to override a default value in a class, but I don't have time to investigate this (not to mention all the tests that need to be changed), so I have implemented as you suggested.